app/views/idv/address/new.html.erb

```diff
@@ -27,7 +27,7 @@
                            input_html: { value: @pii['address2'] } %>
     <div class="margin-bottom-4">
       <%= f.input :city,  label: t('idv.form.city'), required: true, maxlength: 255, wrapper: false,
-                          input_html: { aria: { invalid: false }, class: 'usa-input', value: @pii['city'] } %>
+                          input_html: { aria: { invalid: false }, value: @pii['city'] } %>
```

Comment:
This styling was making the city input text smaller than the other inputs:

<details>

![localhost_3000_verify_address](https://user-images.githubusercontent.com/9039672/177223150-f6cf1b6e-261e-4736-a0f3-0253d9ec6426.png)

</details>